src/subscription_manager/logutil.py

```diff
@@ -63,18 +63,15 @@ def filter(self, record):
         return self.on
 
 
-# NOTE: python 2.6 and earlier versions of the logging module
-#       defined the log handlers as old style classes. In order
-#       to use super(), we also inherit from 'object'
-class RHSMLogHandler(logging.handlers.RotatingFileHandler, object):
-    """Logging Handler for /var/log/rhsm/rhsm.log"""
-    def __init__(self, *args, **kwargs):
-        try:
-            super(RHSMLogHandler, self).__init__(*args, **kwargs)
-        # fallback to stdout if we can't open our logger
-        except Exception:
-            logging.StreamHandler.__init__(self)
-        self.addFilter(ContextLoggingFilter(name=""))
+def RHSMLogHandler(self, *args, **kwargs):
```

Comment:
I like the approach of using a factory method, but `self` argument should be removed.

Reply:
How could i not see this. Will be fixed in a Minute.